src/logic/operate.js

```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const bigOne = Big(numberOne);
+  const bigTwo = Big(numberTwo);
+  switch (operation) {
+    case '-':
+      return bigOne - bigTwo;
```

Comment:
Good job getting this far.
The way you used the new `big.js` I am curious about it.
kindly check out what we have on the doc.
`https://mikemcl.github.io/big.js/`
Example:
`bigOne.minus(bigTwo)`